tools/build.py

```diff
@@ -485,9 +486,7 @@ def build_connect_plugin(args):
                     )
 
         logging.info('Creating archive')
-        archive_path = os.path.join(
-            BUILD_PATH, '{0}-{1}'.format(PROJECT_NAME, CONNECT_PLUGIN_VERSION)
-        )
+        archive_path = os.path.join(BUILD_PATH, os.path.basename(STAGING_PATH))
```

Comment:
Don't we need the connect_plugin_version? I see that is left unused just for logs in the code then?

Reply:
The version number will come from the staging path, so it is preserved. Just wanted to make sure ZIP and the build folder names are 100% aligned

Reply:
Worth removing the CONNECT_PLUGIN_VERSION then as it seems that is not used in the file anymore?

Reply:
Yes, this is just logged now.

What we should perhaps do in CI is check that the plugin version in __version__.py aligns with the tag, so you do no mistakes when creating the manual release? Same goes for libraries.